mlflow/tracking/criteo_authentication.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+from typing import Any
+import subprocess
+
+import requests
+
+from mlflow.store.tracking.rest_store import RestStore
+from mlflow.utils.rest_utils import MlflowHostCreds
+from mlflow.tracking._tracking_service.utils import (
+    _tracking_store_registry,
+    _TRACKING_TOKEN_ENV_VAR,
+)
+
+
+def get_tracking_server_uri() -> str:
+    env = os.getenv("CRITEO_ENV", "preprod")
+    return "https://mlflow.par." + env + ".crto.in"
```

Comment:
Could be dev. I suggest more `domain = "prod" if env == "prod" else "preprod"`

Reply:
Done